cpnest/NestedSampling.py

```diff
@@ -234,7 +234,7 @@ def consume_sample(self):
         logLtmp = []
         for k in self.worst:
             self.state.increment(self.params[k].logL)
-            self.manager.consumer_pipes[k].send(self.params[k])
+            self.manager.consumer_pipes[k].send(self.params[np.random.randint(self.Nlive)])
```

Comment:
Is this a deliberate change or left over from testing?
It will mean that points other than the discarded one will be used to evolve.

Reply:
Actually, it was not supposed to enter this commit, but I think we should implement it on master, see the discussion about copying LPs from Skilling 2004.